script.js

```diff
@@ -1 +1,82 @@
-'use strict'
\ No newline at end of file
+'use strict'
+
+
+const bodySelect = document.querySelector('body')
+const secretN = document.querySelector('.secret')
+const userInput = document.querySelector('.userInput')
+const btnCheck = document.querySelector('.checkBtn')
+const btnReplay = document.querySelector('.replayBtn')
+const position = document.getElementById('position')
+const scoreSelect = document.getElementById('score')
+const highScoreSelect = document.getElementById('highScore')
+
+let secretNumber
+let score
+let highscore = 0;
+
+const setup = function (){
+    secretNumber = Math.trunc(Math.random() * 20) + 1
+    score = 20
+}
+
+setup()
+
+userInput.addEventListener('input', function() {
+if (userInput.value.length === 0) {
+    btnCheck.disabled = true
+} else if (Number(userInput.value) < 1 || Number(userInput.value) > 20){
+    btnCheck.disabled = true
+} else {
+    btnCheck.disabled = false
+}
+})
```

Comment:
# Optimized Repeated Access 

You are accessing the attribute `value` of `userInput` 3 times in a row, maybe you should bind it to a variable first then do your thing.
## `userInput.value`
```js
userInput.addEventListener('input', function() {
let userValue = userInput.value

if (userValue.length === 0) {
    btnCheck.disabled = true
else if (Number(userInput.value) < 1 || Number(userInput.value) > 20){
    btnCheck.disabled = true
} else {
    btnCheck.disabled = false
}
})
```
## `Number(userInput.value)`

```js
userInput.addEventListener('input', function() {
let userValue = userInput.value
const numericValue = Number(userValue)
if (userValue.length === 0) {
    btnCheck.disabled = true
} else if (numericValue < 1 || numericValue > 20){
    btnCheck.disabled = true
} else {
    btnCheck.disabled = false
}
})
```
# More simple if else block
 
I think the first `if` is useless, because the test performed by the `else if` is doing everything you want. 
I could be wrong though.

```js
userInput.addEventListener('input', function() {
let userValue = userInput.value
const numericValue = Number(userValue)
if (numericValue < 1 || numericValue > 20){
    btnCheck.disabled = true
} else {
    btnCheck.disabled = false
}
})
```

Once you see it like this you may start thinking to re-write the whole thing like this:

```js
userInput.addEventListener('input', function() {
    const userNumber = Number(userInput.value)
    btnCheck.disabled = userNumber > 20 || userNumber < 1
})
```